content/blog/getting-started-to-generate-dscc-api-client-libraries-for-python-using-openapi-generator.md

```diff
@@ -0,0 +1,217 @@
+
+OpenAPI Generator tool is available in various form of applications that meet the user's familiarity. The OpenAPI Generator website provides 4 different set of applications:
+
+* A downloadable and executable **JAR** file that can be executed using **Java Run Time tool**.
+* **Docker** image that can be executed using the docker engine.
+* Dependencies in **Maven** and **Gradle** projects that can be used for building automation.
+* Node package manager (**npm**) package wrapper.
+
+Those applications and the information about using the OpenAPI generator is available as the Readme section in the GitHub page from the following URL: <https://github.com/OpenAPITools/openapi-generator> as shown below:
+
+![](/img/openapi-generator.png "OpenAPI Generator GitHub Page")
+
+The key information in this GitHub website that will be useful and important is the latest stable version number that can be used for the conversion. This version is available at the right column of this webpage as shown below:
+
+![](/img/openapi-generator-version-locatoin.png "Stable version for the openAPI generator project")
+
+In this blog, let's look at the painless and simplest application of the openAPI generator, that is using the JAR file. Actually using JAR file doesn't require installation at all. The JAR file can be downloaded and executed directly from the command lines. The requirement for executing the JAR file is that your workstation must be deployed with JAVA runtime 8 (JRE) at a minimum. 
+
+For more information about the deployment of JAVA runtime executables based on the operating system of your workstation, please take a look at the installation page from the JAVA website: <https://www.java.com/en/download/help/download_options.html>
+
+The JAR file for this openAPI generator is available at Maven.org. You can download it from the following location: <https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/5.4.0/openapi-generator-cli-5.4.0.jar>. Below , you will find the syntax required to download the openAPI generator JAR files from the corresponding workstation:
+
+**For Mac/Linux users:**
+
+```shell
+~$ wget https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/5.4.0/openapi-generator-cli-5.4.0.jar -O openapi-generator-cli.jar
+```
+
+**For Windows users:**
+
+```powershell
+> Invoke-WebRequest -OutFile openapi-generator-cli.jar https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/5.4.0/openapi-generator-cli-5.4.0.jar
+```
+
+**Note:** The version of OpenAPI generator which is shown above is 5.4.0 (current version as of March 2022). Please keep in mind that in future, the location of the JAR file will change based on a new code version.  Please take a look at the figure above to obtain the latest version number, and modify the path to download the latest open-api-generator-cli JAR file.
```

Comment:
Recommend rewriting Note:
The OpenAPI Generator version shown above is 5.4.0 (current version as of March 2022). Please keep in mind that, in the future, the location of the JAR file will change once the new version is released. Please take a look at the figure... (the rest remains the same)